src/components/Footer.jsx

```diff
@@ -0,0 +1,13 @@
+import "./Footer.css"
+
+
+const Footer = () => {
+    return (
+        <div className="footer-music">
+            <p>Made by: Erika Olsson</p>
+            <p>Technigo / Web Development Boot Camp</p>
+        </div>
```

Comment:
remember to use semantic elements even though we're working in React (<footer> is better than a <div>)